targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Plugins/PlayFab/Source/PlayFabCommon/Private/TestFramework/PlayFabTestRunner.cpp.ejs

```diff
@@ -94,7 +94,7 @@ void IPlayFabTestRunner::Run(const float InDeltaTime)
         {
             numberOfTests++;
 
-            UE_LOG(LogPlayFabTests, Log, TEXT("Starting test: %s"), *CurrentTestData->testName);
+            UE_LOG(LogPlayFabCommon, Log, TEXT("Starting test: %s"), *CurrentTestData->testName);
```

Comment:
This is slightly confusing to me.
You are adding LogPlayFabTests below, but you're removing multiple uses of it up here.
Can you clarify that?